public/charting_library/ar-tv-chart.287b0032.html

```diff
@@ -0,0 +1 @@
rket_type} cfd":"عقود فروقات ‎{market_type}‎",Light_colorThemeName:"فاتح",Dark_colorThemeName:"داكن","Save Theme As":"احفظ النسق كـ :","Theme name":"أسم القالب","Color Theme '__themeName__' already exists. Do you really want to replace it?":"نسق الألوان ‎__themeName__‎موجود بالفعل. هل ترغب حقًا في استبداله؟","Do you really want to delete Color Theme '{name}' ?":"هل تريد فعلا مسح مظهر اللون ''{name}' ؟",Normal:"عادي","Dashed Line":"خط متقطع","Dotted Line":"خط منقط",Thickness:"السُّمْك","Manage layout drawings":"إدارة رسومات التنسيق","{symbolsCount} symbol_symbols_and_drawings_count_0":"‎{symbolsCount}‎ رمز","{symbolsCount} symbol_symbols_and_drawings_count_1":"‎{symbolsCount}‎ رمز","{symbolsCount} symbol_symbols_and_drawings_count_2":"‎{symbolsCount}‎ رمز","{symbolsCount} symbol_symbols_and_drawings_count_3":"‎{symbolsCount}‎ رموز","{symbolsCount} symbol_symbols_and_drawings_count_4":"‎{symbolsCount}‎ رمزاً","{symbolsCount} symbol_symbols_and_drawings_count_5":"‎{symbolsCount}‎ رمزاً","with {drawingsCount} drawing_symbols_and_drawings_count_0":"مع ‎{drawingsCount}‎ رسم","with {drawingsCount} drawing_symbols_and_drawings_count_1":"مع ‎{drawingsCount}‎ رسم","with {drawingsCount} drawing_symbols_and_drawings_count_2":"مع ‎{drawingsCount}‎ رسم","with {drawingsCount} drawing_symbols_and_drawings_count_3":"مع ‎{drawingsCount}‎ رسوم","with {drawingsCount} drawing_symbols_and_drawings_count_4":"مع ‎{drawingsCount}‎ رسماً","with {drawingsCount} drawing_symbols_and_drawings_count_5":"مع ‎{drawingsCount}‎ رسماً","{drawingsCount} drawing_0":"‎{drawingsCount}‎ رسم","{drawingsCount} drawing_1":"‎{drawingsCount}‎ رسم","{drawingsCount} drawing_2":"‎{drawingsCount}‎ رسم","{drawingsCount} drawing_3":"‎{drawingsCount}‎ رسوم","{drawingsCount} drawing_4":"‎{drawingsCount}‎ رسماً","{drawingsCount} drawing_5":"‎{drawingsCount}‎ رسماً","Remove all drawings for this symbol":"إزالة كافة الرسومات لهذا الرمز","No drawings yet":"لا يوجد رسوم حتى الآن","Remove all line tools for ":"حذف كافة خطوط الأدوات لـ ","Object tree":"شجرة الكائنات","Create a group of drawings":"أنشئ مجموعة من الرسومات","Clone, Copy":"استنساخ ، نسخ","Move to":"حرك إلى","New pane above":"جزء جديد أعلاه","New pane below":"جزء جديد أدناه","Group is hidden on current interval":"المجموعة مخفية في الفاصل الزمني الحالي","Drawing is hidden on current interval":"الرسم مخفي على الفاصل الزمني الحالي","Move objects":"تحريك العناصر",objects:"كائنات","Remove objects":"حذف العناصر","Add Symbol":"إضافة رمز","Confirm Inputs":"تأكيد المدخلات","Chart settings":"إعدادات الرسم البياني","Apply to all":"تفعيل على الكل",Solid:"ثابت",Gradient:"متدرج","No results found":"لا توجد نتائج","Add Custom Color_Color Picker":"إضافة لون مخصص","Opacity_Color Picker":"الشفافية","Add_Color Picker":"إضافة","Remove color":"إزالة اللون","Please enter the right date format yyyy-mm-dd":"الرجاء إدخال التاريخ بالشكل الصحيح يوم- شهر- سنة","Please enter the right time format hh:mm":"الرجاء إدخال الوقت بالشكل الصحيح ساعة: دقيقة"}</script></body></html>
```

Comment:
Hmm idk about this.

Looks a little sketchy, embedding a bundle directly and cross origin anonymous